screenpy_selenium/resolutions/__init__.py

```diff
@@ -3,20 +3,29 @@
 """
 
 from .is_clickable import IsClickable
+from .is_invisible import IsInvisible
+from .is_present import IsPresent
 from .is_visible import IsVisible
 
 # Natural-language-enabling syntactic sugar
 IsEnabled = Enabled = Clickable = IsClickable
 IsDisplayed = Displayed = Visible = IsVisible
-
+IsNotDisplayed = NotDisplayed = Invisible = IsInvisible
+Present = IsPresent
```

Comment:
I appreciate you continuing the alphabetical order of the sugar. :)

Reply:
I think black does that automatically, doesn't it?